CHANGELOG.md

```diff
@@ -75,6 +75,13 @@
 - Bumped inspect from 3.1.2 to 3.1.3
   [#8589](https://github.com/Kong/kong/pull/8589)
 
+
+### Changes
+##### Configuration
+
+- Change the default of `lua_ssl_trusted_certificate` to `system`.
+  [#8602](https://github.com/Kong/kong/pull/8602)
```

Comment:
Include instructions about how to avoid problems with the change:

```suggestion
- Change the default of `lua_ssl_trusted_certificate` to `system`.
  [#8602](https://github.com/Kong/kong/pull/8602). If you are upgrading from 2.x and want this variable to keep
  working as before, please manually set it to `NONE` before upgrading. 
```

I am not 100% sure that the instructions are correct, please double check.